Bounce/dist/metadata.json

```diff
@@ -11,11 +11,12 @@
     "description": {
         "en": "Bounce Radio is a 24/7 international hit music station that provides hit music to everyone. Made by gamers for gamers.",
         "nl": "Bounce Radio is een 24/7 internationaal hitmuziekstation dat iedereen hitmuziek biedt. Gemaakt door gamers voor gamers."
```

Comment:
Comma at the end of nl description because you put another one below it.

Reply:
fixed.